### ISSUE TITLE
DeepFake Test Fail

### ISSUE THREAD

**Issue by miracleyoo** [PeterWang512/CNNDetection#3]
Hello!

I noticed this great repo when I'm working on Kaggle deepfake competition. I tried to test this model, which is supposed to be generalizable, on some **training** set. But the model will output near 0% for both fake and real images.

I tried both whole frames and only cropped faces as input, the output keeps 0. In order to make sure I load and use the model properly, I tested on the example fake and real images in this repo. They work properly, the outputs are 1 and 0.

![image](https://user-images.githubusercontent.com/15561716/76267099-089a9980-6240-11ea-8a8b-224d66dd5177.png)

Here is an example of the deepfake video:

![image](https://user-images.githubusercontent.com/15561716/76267021-c4a79480-623f-11ea-94bf-0f4048d16010.png)

It is apparently a fake face, but the output keeps 0. I'm a little confused about this. 

Thanks a lot!

**Comment by PeterWang512** [comment]
Would you mind checking that if the dimension of the detected face is less than 256 pixels? For the deepfake images, we noticed that most crops are less than 256 pixels, which are too small for our model's input, so we resized all detected face into 256 pixels then pass them into the model to get our results.

**Comment by yangcf10** [comment]
Hi, I am facing similar issues. 
When dealing with deepfake video, I first crop the detected face region (a typical size is 110x110), and then use this transform:
trans = transforms.Compose([
    transforms.Resize(256),
    transforms.CenterCrop(224),
    transforms.ToTensor(),
    transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
])

However, the output probability is almost zero all the time. I am not sure which part I did wrong.

**Comment by PeterWang512** [comment]
This is interesting, thanks for letting us know! Would you mind testing the raw frames in the Faceforensics++ dataset and see if it works? We were testing on the validation set of Faceforensics++ (raw frames), and cropped the face using faced (https://github.com/iitzco/faced) to get our results in the paper. I'll test our model on the compressed version (c23, c40) and see if it works as well. If it doesn't, I'll suspect our model does not generalize to MPEG artifacts, which were not included in our data augmentation.
